src/main/resources/static/index.js

```diff
@@ -2,63 +2,63 @@
 function initMap() {
 
     // The location of UC
-    const uc = { lat: 39.130146, lng: -84.518982 };
+    const UC = { lat: 39.130146, lng: -84.518982 };
     //Mac Shack
-    const mac = { lat: 39.128246, lng: -84.517241 };

```

Comment:
According to javascript best practices provided by Google and GitHub, constants should be in all caps with underscores for multi worded names.